tasks/get_current_archive_list/get_current_archive_list.py

```diff
@@ -202,11 +202,13 @@ def create_job(
                     }
                 ],
             )
+            for sql_result in rows.mappings():
+                job_id = sql_result["id"]
```

Comment:
According to docs, `MappingResult` still has `fetchone` https://docs.sqlalchemy.org/en/20/core/connections.html#sqlalchemy.engine.MappingResult
Cleaner than looping for property sets.

Reply:
It would be best to look into this in a future task.
https://bugs.earthdata.nasa.gov/browse/ORCA-650

Reply:
I added the two places I saw to the card. If you have any others, let's make sure they're documented.

Reply:
added